common/constants.go

```diff
@@ -324,3 +325,41 @@ const (
 	ExternalCommandResponseTopicPrefixKey = "CommandResponseTopicPrefix"
 	ResponseTopicPrefixKey                = "ResponseTopicPrefix"
 )
+
+// MessageBus Topics
+const (
+
+	// Common Topics
+	DefaultBaseTopic    = "edgex"     // Used if the base topic is not specified in MessageBus configuration
+	EventsPublishTopic  = "events"    // <ServiceType>/<DeviceServiceName>/<ProfileName>/<DeviceName>/<SourceName> are appended
+	ResponseTopic       = "response"  // <ServiceName>/<RequestId> are prepended
+	MetricsPublishTopic = "telemetry" // <ServiceName>/<MetricName> are prepended
+
+	// Core Data Topics
+	CoreDataEventSubscribeTopic = "events/device/#"
+
+	// Core Command Topics
+	CoreCommandDeviceRequestPublishTopic  = "device/command/request" // <DeviceServiceName>/<DeviceName>/<CommandName>/<CommandMethod> are appended
+	CoreCommandRequestSubscribeTopic      = "core/command/request/#"
+	CoreCommandQueryRequestSubscribeTopic = "core/commandquery/request/#"
+
+	// Command Client Topics
+	CoreCommandQueryRequestPublishTopic = "core/commandquery/request" // <deviceName>|all is prepended
+	CoreCommandRequestPublishTopic      = "core/command/request"      // <DeviceName>/<CommandName>/<CommandMethod> are appended
+
+	// Core Metadata Topics
+	CoreMetadataSystemEventPublishTopic = "system-events" // <SourceServiceName>/<SystemEventType>/<SystemEventAction><OwnerServiceName>/<ProfileName>
+
+	// Support Notifications
+	// No Topics Yet
+
+	// Support Scheduler
+	// No Topics Yet
+
+	// Device Services Topics
+	DeviceServiceCommandRequestSubscribeTopic = "device/command/request"               // <DeviceServiceName>/# is appended <
+	DeviceServiceSystemEventSubscribeTopic    = "system-events/core-metadata/device/+" // <DeviceServiceName>/# is appended
```

Comment:
suggest renaming to `DeviceServiceMetadataSystemEventSubscribeTopic` in case there will be other sources of system events subscribed by Device Service.

Reply:
How about `DeviceSystemEventsSubscribeTopic`  since this is specific to device system events.

Reply:
It's for Device, Device Profile, and Provision Watcher System Event, so I prefer to call it MetadataSystemEvent

Reply:
Oh, the value will become `system-events/core-metadata/+/+`, and Ginny is working on the implementation in device-sdk-go

Reply:
The trailing `/+/+` can just be `/#`
I have update the constant to be `MetadataSystemSubscribeEvent              = "system-events/core-metadata/#"`